tests/src/test/java/alluxio/master/journal/ufs/UfsJournalConfTest.java

```diff
@@ -23,7 +23,7 @@
 /**
  * Tests for {@link UfsJournal}.
  */
-public class UfsJournalTest {
+public class UfsJournalConfTest {
```

Comment:
make this a unit test?

Reply:
Will do in a follow up (covered in the package-info todo).

Reply:
sg